lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -668,7 +660,7 @@ def push_student_forums_data_to_s3(_xmodule_instance_args, _entry_id, course_id,
     return _push_csv_responses_to_s3(collect_student_forums_data, u'student_forums', course_id, action_name)
 
 
-def _push_csv_responses_to_s3(csv_fn, filename, course_id, action_name):
+def _push_csv_responses_to_s3(csv_fn, filename, course_id, action_name, include_email=None):
```

Comment:
Is there any way we can avoid changing this method's header to include include_email?  Lots of other reports use _push_csv_responses_to_s3, so changing its input arguments underneath them is not a great option.  I would almost rather separate out a collect_anonymous_ora2_data and collect_email_ora2_data method so that the include_email logic could be moved out of this method and included with the existing conditional in push_ora2_responses_to_s3.  Another option would be to change include_email to a more general parameter like kwargs with the same None default value.  Then, you could check to see if kwargs is None instead of include_email and pass it to the csv_fn if not.  This also would make the parameter available if others want to parameterize their data collection methods in the future.  I'm not enthusiastic about adding this parameter, though.